scaffoldmaker/meshtypes/meshtype_3d_sphereshell1.py

```diff
@@ -311,7 +323,10 @@ def generateBaseMesh(region, options):
                 for e1 in range(elementsCountAround):
                     va = e1
                     vb = (e1 + 1)%elementsCountAround
-                    eft1 = tricubichermite.createEftShellApexBottom(va*100, vb*100)
+                    if useCubicHermiteThroughWall:
```

Comment:
These 4 lines should just be:

    eft1 = eftfactory.createEftShellPoleBottom(va*100, vb*100)

as the same code is called in both cases.